src/components/button/button.tsx

```diff
@@ -0,0 +1,30 @@
+import cn from "classnames";
```

Comment:
I suggest to use `clsx` instead of `classnames`.
[Open NPM](https://www.npmjs.com/package/clsx)